### PR TITLE
add bert-large-uncased-whole-word-masking-squad-embedding-0001 model and code for accuracy calculation on squad 1.1 dev dataset

### PR DESCRIPTION
**Pull request by krodyush**

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins ok to test

File: tools/accuracy_checker/accuracy_checker/annotation_converters/squad_emb.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
+"""
+Copyright (c) 2019 Intel Corporation
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/squad_emb.py:2]:
```suggestion
Copyright (c)  2018-2020 Intel Corporation
```

File: tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py [hunk 2]
```diff
@@ -255,14 +255,15 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
 
                 p_mask = self.fill_p_mask(span, len(truncated_query), sequence_added_tokens, cls_index)
                 idx = example_index
-                identifier = ['input_ids_{}'.format(idx), 'input_mask_{}'.format(idx), 'segment_ids_{}'.format(idx)]
+                identifier = ['input_ids_{}'.format(idx), 'input_mask_{}'.format(idx), 'segment_ids_{}'.format(idx), 'position_ids_{}'.format(idx)]
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py:258]: please remove changes not connected with this model support

Comment (krodyush) [reply tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py:258]: it is connected to this model support

File: models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/accuracy-check.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+models:
+  - name: bert-large-uncased-whole-word-masking-squad-embedding-0001
+    launchers:
+      - framework: dlsdk
+        allow_reshape_input: True
+        adapter:
+            type: bert_question_answering_embedding
+            embedding: "embedding"
```
Comment (Anna-Mironova) [inline models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/accuracy-check.yml:8]: Model has no 'embedding' output

```suggestion
            embedding: "3157/Fused_Add_"
```

File: tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py (hunk 2 above)
Comment (krodyush) [reply tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py]: removed

File: tools/accuracy_checker/accuracy_checker/annotation_converters/squad_emb.py [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+            squad['data'] = pool.starmap(
+                encode_squad_article,
+                zip(squad['data'], [self.vocab] * N, [self.lower_case] * N)
+            )
+
+        pad = [self.vocab["[PAD]"]]
+        cls = [self.vocab["[CLS]"]]
+        sep = [self.vocab["[SEP]"]]
+
+        index_ref=[0]
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/squad_emb.py:156]:
```suggestion
        index_ref = [0]
```

**Comment by eaidova** [comment]
@krodyush please describe new adapter, converter and metric in appropriate readme files

**Comment by eaidova** [comment]
@krodyush  please add bert-large-uncased-whole-word-masking-squad-embedding-0001 into [index.md](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/models/intel/index.md)

**Comment by eaidova** [comment]
@eizamaliev please take a look on model description

**Review by eaidova — APPROVED** [review]

File: models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+# bert-large-uncased-whole-word-masking-qcemb-0001
```
Comment (eizamaliev) [inline models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md:1]: Name differs from directory and file names.

File: models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+[BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805),
+[HuggingFace's Transformers: State-of-the-art Natural Language Processing](https://arxiv.org/abs/1910.03771).
+
+Tokenization occurs using the BERT tokenizer (see the demo code for implementation details) and the enclosed `vocab.txt` dictionary file. Input is to be lower-cased before tokenizing.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| GOps              | 246.93 (for [1,384] input size)|
```
Comment (eizamaliev) [inline models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md:20]:
```suggestion
| GOps              | 246.93                |
```
Or comment this specification.

**Review by eizamaliev — CHANGES_REQUESTED** [review]

File: models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md (hunk 6 above)
Comment (krodyush) [reply models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md:20]: the model is used in 2 modes with [1,384] input shape and with [1,32] input shape. So it should be noted for which shape GOps is calculated.

Comment (krodyush) [reply models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md:20]: @eizamaliev what do you mean "comment this specification". can you provide example.

Comment (eaidova) [reply models/intel/bert-large-uncased-whole-word-masking-squad-embedding-0001/description/bert-large-uncased-whole-word-masking-squad-embedding-0001.md:20]: @krodyush , I think @eizamaliev  means that this comment should be provided outside of the table in model description like clarification how metric was calculated (add sentence  `GOps calculated for model with input size [1, 384[` or something simmilar)

**Review by eizamaliev — APPROVED** [review]

**Pull request merged**